build/patches/libsrtp-two-byte-rtp-header-extension-crypto.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a//third_party/libsrtp/srtp/srtp.c b//third_party/libsrtp/srtp/srtp.c
+index 4462501..bfb596e 100644
+--- a//third_party/libsrtp/srtp/srtp.c
++++ b//third_party/libsrtp/srtp/srtp.c
+@@ -1422,7 +1422,7 @@ static srtp_err_status_t srtp_process_header_encryption(
+                 xtn_hdr_data++;
+             }
+         }
+-    } else if ((ntohs(xtn_hdr->profile_specific) & 0x1fff) == 0x100) {
++    } else if ((ntohs(xtn_hdr->profile_specific) & 0x1fff) == 0x1000) {
```

Comment:
Hmm, shouldn't the first ANDed value [be `0xfff0` instead of `0x1fff`](https://github.com/cisco/libsrtp/pull/491/files)? Or is there a reason why this differs from the libsrtp patch?

Reply:
Yeah, looks like I took this from an old patch. Good catch!